README.md

```diff
@@ -130,9 +130,26 @@ spec:
     key: token
 ```
 
-This will fetch the secret, and get the value in `token` and use that to
-authenticate the API call to GitHub.
-
+This will fetch the secret, and get the value in `token` and use that to authenticate the API call to GitHub. The secret may contain multiple values. For example: 
```

Comment:
```suggestion
This will fetch the secret, and get the value in `token` and use that to authenticate the API call to GitHub. The secret may contain multiple values and you can configure which key within the `Secret` by setting the `key` field in the `spec.auth` configuration, this defaults to `token`.
```